access-checkout-react-native-sdk/README.md

```diff
@@ -11,14 +11,23 @@ It is designed to simplify the integration of the following functionality in you
 
 ## Documentation
 
-You can find the detailed documentation explaining how to integrate the SDK and use a session to take a payment starting with the [React Native section](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native) of the [Access Worldpay documentation](https://developer.worldpay.com).
+You can find the detailed documentation explaining how to integrate the SDK and use a session to take a payment starting with the [React Native section](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native) of the [Access Worldpay Checkout documentation](https://developer.worldpay.com).
 
 ## Compatibility
 
 - all `react-native` versions >= `0.61.5`
 - all `react` versions >= `16.9.0`
 - `Cocoapods` only for iOS dependencies
 
+## SAQ-A Compliance
+
+AccessCheckoutTextInput is the name of the new component created to provide SAQ-A compliance when using our React Native SDK.
+It has been designed so that it does not expose any methods or properties to retrieve the text entered by the end user to ensure our merchants applications are SAQ-A compliant.
+
+## AccessCheckoutTextInput
+
+You can find detailed documentation about our new component [AccessCheckoutTextInput](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native/access-checkout-text-input) within the [React Native section](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native) of the [Access Worldpay Checkout documentation](https://developer.worldpay.com).
```

Comment:
Do you mind renaming the `access-checkout-text-input.md` file `optional-configuration.md` to match the naming convention used for Android and iOS, and then change the link here to point to this page?

Reply:
amend path to be /react-native/optional-configuration